core/src/test/scala/com/metamx/tranquility/test/HashPartitionBeamTest.scala

```diff
@@ -37,11 +38,18 @@ class HashPartitionBeamTest extends FunSuite with Matchers
     override def hashCode(): Int = s(0).toInt
   }
 
-  class TestBeam(callback: TestObject => Unit) extends Beam[TestObject]
+  class DroppingBeam() extends Beam[TestObject]
   {
-    override def propagate(events: Seq[TestObject]): Future[Int] = {
+    override def sendBatch(messages: Seq[TestObject]): Future[BitSet] = Future.value(BitSet.empty)
```

Comment:
dropped events should not show up in the BitSet, that's how callers know they've been dropped